docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml

```diff
@@ -41,6 +48,18 @@ code: |
   installer.set_github_auth()
   set_github_auth = True
 ---
+code: |
+  # if vals are made in module using `.content`, they trigger
+  # the YAML logic block repeatedly, causing multiple
+  # new branches
```

Comment:
This comment would be easier if it said something about the alternative. "If these values were in `al_set_up_testing.py` then they would cause unnecessary looping and create multiple unnecessary new branches. So we put them in `al_testing_files_to_push.yml` and set the variables in `al_set_up_testing` like this."

Reply:
(easier for me to parse, at least) I didn't understand it at first.